roles/KubevirtCommonTemplatesBundle/tasks/main.yml

```diff
@@ -1,4 +1,19 @@
 ---
+- name: Create namespace for golden images
```

Comment:
Please move your new tasks under `Set operatorVersion and targetVersion`, because the first task of the playbook should indicate that the operator might be performing an upgrade.